src/tactics/FStar.Tactics.Interpreter.fs

```diff
@@ -504,8 +510,13 @@ let run_tactic_on_ps
     Err.stop_if_err ();
```

Comment:
FYI, this chunk of changes to run_tactic_on_ps will not survive to master. We will have a separate PR into master that revises this and then incorporate those changes here.